fred2/missionsave.cpp

```diff
@@ -4348,3 +4370,15 @@ int CFred_mission_save::save_wings()
 	Assert(count == Num_wings);
 	return err;
 }
+
+// quick helper function to check if a wing is currently defined.
+bool is_wing_defined(char* wing_name)
```

Comment:
You may want to make this `const char *`.

Additionally, if you move the function above `CFred_mission_save::autosave_mission_file`, you won't need the forward declaration.